### DISCUSSION TITLE
Run outside of Docker

### DISCUSSION THREAD

**Discussion by timastana** [microrealestate/microrealestate#196] [Q&A]
How challenging would it be to run the project outside of Docker? What potential difficulties should I expect?

**Comment by camelaissani** [comment]
To run the app without Docker, you will need to
- Install Redis and Mongodb
- Start the node micro services with the right environment variables
- Start the front-ends

Or you can use Docker compose with the new install procedure:

https://github.com/microrealestate/microrealestate?tab=readme-ov-file#self-host-the-application
